app/views/ReportView/components/GenomicSummary/components/TumourSummaryEdit/index.tsx

```diff
@@ -82,18 +91,49 @@ const TumourSummaryEdit = ({
     }
   };
 
-  const handleClose = useCallback((isSaved) => {
+  const handleClose = useCallback(async (isSaved) => {
     if (isSaved) {
+      setIsApiCalling(true);
+      const apiCalls = [];
+
+      if (newMicrobialData) {
+        if (microbial) {
+          apiCalls.push(api.put(`/reports/${report.ident}/summary/microbial/${microbial.ident}`, newMicrobialData, {}));
```

Comment:
Running into problems when the microbial doesn't have an ident the api call will error out
`ttps://iprdev-api.bcgsc.ca/api/reports/9d9db515-fe5f-46c2-b6a8-3d59d90bca8a/summary/microbial/undefined`